hugo-modules/core/layouts/sitemap.xml

```diff
@@ -0,0 +1,39 @@
+{{ printf "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"yes\"?>" | safeHTML }}
+<urlset xmlns="https://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance" xmlns:xhtml="https://www.w3.org/1999/xhtml" xsi:schemaLocation="https://www.w3.org/1999/xhtml https://www.w3.org/2002/08/xhtml/xhtml1-strict.xsd">
+  {{ range .Data.Pages }}
+
+    {{/* Get pages excluded from indexing by Contentful */}}
+    {{ $no_index := false }}
+    {{ with .Params.seo }}
+      {{ $seo := partial "utils/get-data" . }}
+      {{ $no_index = $seo.no_index }}
+    {{ end }}
+
+    {{/* Get page templates that are on the blocklist and should generally not be indexed */}}
+    {{ $blocked := false }}
+    {{ with .Params.sys.content_type }}
+      {{ $blocked = in site.Params.seo_blocklist . }}
+    {{ end }}
```

Comment:
This part will probably need a short documentation.

Alternatively, we could include an example in https://github.com/jungvonmatt/wekit/blob/main/templates/app/config/_default/params.yaml

```
# If a page type is on the seo_blocklist, then associated pages are generally not indexed
seo_blocklist:
# - t-notification
 ```

In order to do it consistently, we would additionally need to include it in in these two files:
- layouts/partials/utils/seo/data.html
- layouts/robots.txt

Reply:
Should I exclude that part here or can you commit the required changes? I'm not entirely sure what to do, honestly.

Reply:
Then I would suggest we remove it for now. The extension for the blocklist is nice, but we can also discuss again if we really want to have it in the default.